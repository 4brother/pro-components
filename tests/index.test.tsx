import { mount } from '@vue/test-utils';
// import BasicLayout from '../src/BasicLayout';

describe('BasicLayout', () => {

  it('ðŸ¥© base use', () => {
    // const wrapper = mount(BasicLayout, { });
    // console.log(wrapper.html());
    // expect(wrapper.html()).toMatchSnapshot();
  });

});
