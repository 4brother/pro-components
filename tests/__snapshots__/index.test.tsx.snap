// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BasicLayout ðŸ¥© base use 1`] = `
<div class="ant-pro-basicLayout ant-pro-basicLayout-side">
  <section class="ant-layout ant-pro-basicLayout">
    <!---->
    <aside class="ant-pro-sider ant-pro-sider-light ant-pro-sider-side ant-layout-sider ant-layout-sider-light" style="flex: 0 0 208px; max-width: 208px; min-width: 208px; width: 208px;">
      <div class="ant-layout-sider-children">
        <div class="ant-pro-sider-logo"><a><img src="https://alicdn.antdv.com/v2/assets/logo.1ef800a8.svg" alt="logo">
            <h1>Pro Tests</h1>
          </a></div>
        <div style="flex: 1; overflow: hidden auto;">
          <ul role="menu" class="ant-pro-sider-menu ant-menu-light ant-menu-root ant-menu ant-menu-inline" style="width: 100%;"></ul>
        </div>
        <div class="ant-pro-sider-links">
          <ul role="menu" class="ant-pro-sider-link-menu ant-menu-light ant-menu-root ant-menu ant-menu-inline">
            <!---->
            <li role="menuitem" style="padding-left: 16px;" class="ant-pro-sider-collapsed-button ant-menu-item"><span role="img" aria-label="menu-fold" class="anticon anticon-menu-fold"><svg class="" data-icon="menu-fold" width="1em" height="1em" fill="currentColor" aria-hidden="true" viewBox="64 64 896 896" focusable="false"><path d="M408 442h480c4.4 0 8-3.6 8-8v-56c0-4.4-3.6-8-8-8H408c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8zm-8 204c0 4.4 3.6 8 8 8h480c4.4 0 8-3.6 8-8v-56c0-4.4-3.6-8-8-8H408c-4.4 0-8 3.6-8 8v56zm504-486H120c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-56c0-4.4-3.6-8-8-8zm0 632H120c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-56c0-4.4-3.6-8-8-8zM115.4 518.9L271.7 642c5.8 4.6 14.4.5 14.4-6.9V388.9c0-7.4-8.5-11.5-14.4-6.9L115.4 505.1a8.74 8.74 0 000 13.8z"></path></svg></span>
              <!---->
            </li>
          </ul>
        </div>
      </div>
      <!---->
    </aside>
    <section class="ant-layout" style="position: relative; min-height: 0;">
      <!---->
      <header class="ant-layout-header" style="padding: 0px; height: 48px; line-height: 48px; width: 100%; z-index: 19;">
        <div class="ant-pro-global-header ant-pro-global-header-layout-side">
          <!---->
          <!---->
          <!---->
          <div style="flex: 1;">
            <!---->
          </div>
          <div class="custom-header-right-content"><span>custom-right-content</span></div>
        </div>
      </header>
      <main class="ant-layout-content ant-pro-basicLayout-content ant-pro-basicLayout-has-header" ischildrenlayout="false" style="min-height: 300px;">
        <div>content</div>
      </main>
      <div>custom-footer</div>
    </section>
  </section>
</div>
`;
